webapp/src/pages/EditAvatarPage/EditAvatarPage.css

```diff
@@ -0,0 +1,38 @@
+.edit-avatar-page {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  gap: 3rem var(--column-gap);
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto 1fr auto;
```

Comment:
I think Ii will revert this to use `display: flex`  instead of `grid` and `justify-content: space-between;` in mobile as well. Let me know if we all agree or if you have any other comments. If I don't, I'll make this change and merge into main tomorrow evening.

Reply:
For mobile looks much better as a grid, than as a flex with space between, so I'll not change it. I'm following figma.

Reply:
Ok, for is OK, both versions, grid and flex box